docs/source/tensors.rst

```diff
@@ -37,6 +37,8 @@ quantized 8-bit integer (unsigned)      ``torch.quint8``
 quantized 8-bit integer (signed)        ``torch.qint8``
 quantized 32-bit integer (signed)       ``torch.qint32``
 quantized 4-bit integer (unsigned) [3]_ ``torch.quint4x2``
+8-bit floating point, e4m3 [5]_         ``torch.float8_e4m3fn`` (limited support)
+8-bit floating point, e5m2 [5]_         ``torch.float8_e5m2`` (limited support)
```

Comment:
can we also add the 'nz' variants

Reply:
the variants above are further along in their lifecycle as we actually have e2e things working throughout the stack and it's crossing the threshold of "this is useful".  I don't think the nz variants are quite there yet.

Reply:
@jeffdaily Is that a fair characterization, that we don't want them documented here

Reply:
I'm going to land this, we can always follow up separately on the nz types

Reply:
AFAIK pytorch today doesn't have anything implemented to demonstrate the fnuz types on hardware.  We can follow up separately, even after #115214, once there might be some unit tests or other e2e demonstration.

OTOH, #115214 adds limited op support in pytorch for the fnuz types.